src/AssemblySharedInfoGenerator/AssemblyInfoGenerator.csproj

```diff
@@ -0,0 +1,34 @@
+<Project>
+  <!-- SDK style projects need to do manual imports of sdk props/targets in order to compile t4 templates on each build -->
+  <Import Project="Sdk.props" Sdk="Microsoft.NET.Sdk" />
+  <ImportGroup Label="PropertySheets">
+    <Import Project="$(SolutionDir)Config\CS_SDK.props" />
+  </ImportGroup>
+  <PropertyGroup>
+    <TransformOnBuild>true</TransformOnBuild>
+    <OverwriteReadOnlyOutputFiles>true</OverwriteReadOnlyOutputFiles>
+    <TransformOutOfDateOnly>false</TransformOutOfDateOnly>
+    <ProjectGuid>{133FC760-5699-46D9-BEA6-E816B5F01016}</ProjectGuid>
+    <OutputType>Library</OutputType>
+    <AppDesignerFolder>Properties</AppDesignerFolder>
+    <RootNamespace>AssemblyInfoGenerator</RootNamespace>
+    <AssemblyName>AssemblyInfoGenerator</AssemblyName>
+    <OutputPath>bin\$(Configuration)\</OutputPath>
+    <EnableDefaultCompileItems>False</EnableDefaultCompileItems>
+  </PropertyGroup>
+  <ItemGroup>
+    <None Include="AssemblySharedInfo.cs">
+      <AutoGen>True</AutoGen>
+      <DesignTime>True</DesignTime>
+      <DependentUpon>AssemblySharedInfo.tt</DependentUpon>
+    </None>
+  </ItemGroup>
+  <ItemGroup>
+    <None Include="AssemblySharedInfo.tt">
+      <Generator>TextTemplatingFileGenerator</Generator>
+      <LastGenOutput>AssemblySharedInfo.cs</LastGenOutput>
+    </None>
+  </ItemGroup>
```

Comment:
This project will not compile any source code  ... All it will do is to generate the AssemblySharedInfo.cs form its t4 counterpart.
The AssemblySharedInfo.cs file will be referenced by all the other Dynamo projects (as linq)